myapp/api/models/users.py

```diff
@@ -8,8 +8,8 @@ class CustomUserBase(models.Model):
     user_id = models.UUIDField(
         primary_key=True, default=uuid.uuid4, editable=False, unique=True
     )
-    first_name = models.CharField(max_length=65, blank=True, null=True)
-    preferred_name = models.CharField(max_length=65)
+    first_name = models.CharField(max_length=65)
+    preferred_name = models.CharField(max_length=65, blank=True, null=True)
```

Comment:
Is there a reason why this can now be blank. From my understanding we populate this field with first name if preferred name wasn't entered, shouldn't that mean that preferred name should still always have a value?